firewall/firewall_interfaces.go

```diff
@@ -4,11 +4,20 @@ package firewall
 type QueryHandlerInterface interface {
 	CheckQuery(sqlQuery string) error
 
+	Refresh()
```

Comment:
what this method should do in handlers?

Reply:
as I see below it reset state of handler. Maybe better to rename to `Reset()`?

Reply:
Agree. Changed